tests/neg-macros/reflect-inline/test_2.scala

```diff
@@ -0,0 +1,6 @@
+import api._
+
+class Test {
+  val a: String = "5"
+  a.stripMargin2 // error: `a should be a known constant
```

Comment:
@odersky, could you have a look at this one? I could not figure out what change in this PR affected this.

In master it fails with `value stripMargin2 is not a member of String` because `a` is not and `inline` `String`. But for some reason in this PR `stripMargin2` is accepted as a member of `a` and it does not check and skips the check that `a` should be an inline parameter. It should fail before inlining the code.